src/AppKit/NSAccessibility.cs

```diff
@@ -35,11 +39,11 @@ public partial class NSAccessibility {
 		[Mac (10, 10)]
 #endif
 		[DllImport (Constants.AppKitLibrary)]
-		static extern CGRect NSAccessibilityFrameInView (NSView parentView, CGRect frame);
+		static extern CGRect NSAccessibilityFrameInView (NativeHandle parentView, CGRect frame);
 
 		public static CGRect GetFrameInView (NSView parentView, CGRect frame)
 		{
-			return NSAccessibilityFrameInView (parentView, frame);
+			return NSAccessibilityFrameInView (parentView.Handle, frame);
```

Comment:
This avoids a potential NullReferenceException:

```suggestion
			return NSAccessibilityFrameInView (parentView.GetHandle (), frame);
```